functional-test/src/test/java/org/zanata/feature/document/UploadTest.java

```diff
@@ -90,45 +90,6 @@ public void before() {
     }
 
     @Test(timeout = ZanataTestCase.MAX_SHORT_TEST_DURATION)
-    @Category(BasicAcceptanceTest.class)
-    public void uploadedDocumentIsInFilesystem() {
-        File originalFile =
-                testFileGenerator.generateTestFileWithContent(
-                        "uploadedDocumentIsInFilesystem", ".txt",
-                        "This is a test file");
-        String testFileName = originalFile.getName();
-
-        VersionDocumentsTab versionDocumentsTab = new ProjectWorkFlow()
-                .goToProjectByName("uploadtest")
-                .gotoVersion("txt-upload")
-                .gotoSettingsTab()
-                .gotoSettingsDocumentsTab()
-                .pressUploadFileButton()
-                .enterFilePath(originalFile.getAbsolutePath())
-                .submitUpload()
-                .clickUploadDone();
-
-        assertThat(new File(documentStorageDirectory).list().length)
-                .isEqualTo(1)
-                .as("There is only one uploaded source file");
-
-        File newlyCreatedFile = new File(documentStorageDirectory,
-                testFileGenerator
-                        .getFirstFileNameInDirectory(documentStorageDirectory));
-
-        assertThat(testFileGenerator.getTestFileContent(newlyCreatedFile))
-                .isEqualTo("This is a test file")
-                .as("The contents of the file were also uploaded");
-        VersionDocumentsPage versionDocumentsPage = versionDocumentsTab
-                .gotoDocumentTab()
-                .waitForSourceDocsContains(testFileName);
-
-        assertThat(versionDocumentsPage.sourceDocumentsContains(testFileName))
-                .isTrue()
-                .as("Document shows in table");
-    }
```

Comment:
I discussed them with @djansen-redhat because they are testing what happens in the file system, not how the server behaves.  We already have tests that ensure we can upload and download.  Whether they go to the file system or some sort of database is not something that should break a test.